src/components/modals/Debug.js

```diff
@@ -51,6 +51,7 @@ class Debug extends Component<*, *> {
       for (let x = 0; x < 20; x++) {
         const { address, path } = await getAddress
           .send({
+            derivationMode: 'segwit',
```

Comment:
`account.derivationMode` ?

Reply:
in context of this Debug screen, it seems we only do segwit derivations. https://github.com/LedgerHQ/ledger-live-desktop/blob/7027e6316d5734166c1b8863c4a656791f631830/src/components/modals/Debug.js#L49